tests/storage/test_user_directory.py

```diff
@@ -76,6 +78,19 @@ async def get_users_in_user_directory(self) -> Set[str]:
         )
         return {row["user_id"] for row in result}
 
+    async def get_profiles_in_user_directory(self) -> Dict[str, ProfileInfo]:
```

Comment:
this class is starting to get to the point where docstrings for the methods wouldn't go amiss. Could you add a quick docstring for this method?